### ISSUE TITLE
[request] brew method to install

### ISSUE THREAD

**Issue by MohamadKh75** [tzmax/V2RayXS#26]
Hey

Thanks for your awesome tool! Can you please provide the [Homebrew](https://brew.sh/) cask for this?

**Comment by tzmax** [comment]
I am happy to provide support, I will post in subsequent work

**Comment by MohamadKh75** [comment]
Just a reminder, i guess it's not too hard since we have [v2rayX formula](https://formulae.brew.sh/cask/v2rayx#default)

**Comment by tzmax** [comment]
> Just a reminder, i guess it's not too hard since we have [v2rayX formula](https://formulae.brew.sh/cask/v2rayx#default)

@MohamadKh75 Of course, but it has not been sorted out due to time, if you are willing to help me, I can try to submit a PR

**Comment by ne-ray** [comment]
@tzmax #Homebrew/homebrew-cask/pull/205132
